### PR TITLE
feat: create settings page

### PR DESCRIPTION
**Pull request by Kojo-Brown**

### CODE REVIEW

File: src/app/dashboard/settings/settings.component.ts [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+    this.passwordInfo();
+  }
+
+  private generalInfo() {
+    this.generalInfoForm.patchValue({
+      name: this.authService.loggedInUser()?.user.name,
+      role: capitalizeFirstLetter(this.authService.role() as string),
+      email: this.authService.loggedInUser()?.user.email,
+      phone: '0211113334',
```
Comment (ghost) [inline src/app/dashboard/settings/settings.component.ts:82]: Shouldn't this be user's phone?

File: src/environments/environment.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
 export const environment = {
   production: true,
-  apiUrl: 'https://sef-foundation-backend.onrender.com/',
+  apiUrl: 'http://localhost:3000',
```
Comment (ghost) [inline src/environments/environment.ts:3]: You can change remove this for .development but not here

File: src/app/dashboard/settings/settings.component.ts [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+      twoFactorAuthentication: [''],
+    });
+
+    this.passwordInfo();
+  }
+
+  private generalInfo() {
+    this.generalInfoForm.patchValue({
+      name: this.authService.loggedInUser()?.user.name,
+      role: capitalizeFirstLetter(this.authService.role() as string),
```
Comment (ghost) [inline src/app/dashboard/settings/settings.component.ts:80]: Just wondering if the angular `TitleCasePipe` Can be used here

**Review by ghost — APPROVED** [review]

File: src/app/dashboard/settings/settings.component.ts (hunk 1 above)
Comment (Kojo-Brown) [reply src/app/dashboard/settings/settings.component.ts:82]: We don't have the user's phone in the user's data from authservice. We can get it from the beneficiary but also with that we don't have some of the data there as well

Comment (ghost) [reply src/app/dashboard/settings/settings.component.ts:82]: Just make it empty string or?

Comment (Kojo-Brown) [reply src/app/dashboard/settings/settings.component.ts:82]: This is not the final work. I just wanted you to have the component you asked for

**Pull request merged**